tests/integration/test_simple_thermostat_fmu.py

```diff
@@ -56,7 +56,7 @@ def test_io_with_external_clock(self):
 
         # Attempt to override the measured temp (ie zone temperature),
         # and the setpoint, such that zone temperature is over setpoint.
-        self.alfalfa.setInputs(self.model_id, {"oveWriMeasuredTemp_u": 303.15, "TsetPoint_u": 294.15})
+        self.alfalfa.setInputs(self.model_id, {"oveWriMeasuredTemp_u": 303.15, "oveWriSetPoint_u": 294.15})
```

Comment:
the previous values weren't actually defined in the FMU, but it didn't matter because they weren't needed to trip the control signal for testing.